### ISSUE TITLE
Cosmetic layout issue in display of Backup Contents

### ISSUE THREAD

**Issue by kghandi** [sabeechen/hassio-google-drive-backup#625]
Thank you for a great add-on. It is working great. So this is purely a cosmetic issue. Since some entries in the backup Contents view do not have a subtitle, they have slightly smaller height, causing a gap in the layout. For example, below HA Config entry in the screenshot below. Also notice that there is a different padding between the icons and titles:
 
<img width="684" alt="image" src="https://user-images.githubusercontent.com/6336722/171225603-7fa73fbc-4197-4bc7-bfa3-9423018d6f07.png">

**Comment by sabeechen** [comment]
Hrm you're right about the folder vs addon icon size, I wonder how I never noticed that?  

As for the gap in the layout...
... I'm going to try to say this without sounding crazy....  but I have spent LITERALLY HOURS trying to figure out why that gap appears.  It seems like no matter what I do it still shows up.

Could it actually be because of the height difference?  I've learned from this project that a _true understanding_ of HTML/CSS layout rules requires a greater mind than I have at my disposal.  If you've identified why the layout renders with a gap I'm going be both very grateful and pissed.

**Comment by sabeechen** [comment]
Yup, that was it.  So much wasted time.  Thank you for identifying the issue accurately.

**Comment by kghandi** [comment]
If I add the following css in Chrome DevTools the issue is resolved:

```
.details-contents > .detail-badge.content-badge {
    min-height: 33px
}

.details-contents > .detail-badge.content-badge > .left {
    min-width: 45px
}
```

**Comment by sabeechen** [comment]
I did something similar to fix it, and just released v0.108.0 of the addon.  Thanks for noticing the problem and reporting it!
